nvflare/app_common/workflows/base_model_controller.py

```diff
@@ -139,9 +141,7 @@ def broadcast_model(
         if not blocking and not isinstance(callback, Callable):
             raise TypeError("callback must be defined if blocking is False, but got {}".format(type(callback)))
 
-        if not data:
```

Comment:
Can this break existing cases?

Reply:
Changed the method signature to remove the default None value.